.github/multi-labeler.yml

```diff
@@ -20,26 +22,38 @@ labels:
   - label: 'chore'
     matcher:
       title: '^chore(\(|:)'
-  # "change",
   - label: 'docs'
     matcher:
       title: '^docs(\(|:)'
-  # "style",
   - label: 'refactor'
     matcher:
       title: '^refactor(\(|:)'
-  # "test",
   - label: 'auto'
     matcher:
       title: '^auto(\(|:)'
 
-  # Below are the scopes that we look for in the PR title
+  # additional meta flags
+
+  # note, this does not pick up chained PRs automatically
+  - label: 'stable'
+    matcher:
+      baseBranch: '^stable-.+'
+
+  - label: 'cherry-pick'
+    matcher:
+      title: '(🍒|:cherries:)'
```

Comment:
Could maybe add one for...

```
label: 'feature-branch'
matcher:
  branch: 'feature-.*'
```

Or something like that?  Just a suggestion, though.

Reply:
Good idea, missed that one. Done!